### PR TITLE
docs(pagination): update docs

### LINKED ISSUE #930: [docs] Pagination docs need review/cleanup
Please review sections and ensure docs are complete.

Current tabs: Overview, Code. No content in the Overview tab, currently. “Overview” heading in the Code tab has no additional description beyond the name of the element.  No Guidelines tab.

Tag @coreyvickery for final design review.

### PR DESCRIPTION
**Pull request by eyevana**
Closes #930

## What I did

1. Updated the documentation for the pagination element according to the specs outlined in #930.

## Testing Instructions

1. Review latest docs at https://deploy-preview-993--red-hat-design-system.netlify.app/elements/pagination/ and verify they match the design specs as well as the docs cleanup guide https://github.com/RedHat-UX/red-hat-design-system/wiki/Docs-Cleanup-Q2-2023-Checklist#code-page.

### CODE REVIEW

**Comment by eyevana** [comment]
@coreyvickery according to the docs cleanup [guide](https://github.com/RedHat-UX/red-hat-design-system/wiki/Docs-Cleanup-Q2-2023-Checklist#images), we should also have a `screenshot.svg` as an asset. Is that something that you provide? Or do I have take the screenshot.png file and somehow convert it into an SVG?

**Comment by coreyvickery** [comment]
> @coreyvickery according to the docs cleanup [guide](https://github.com/RedHat-UX/red-hat-design-system/wiki/Docs-Cleanup-Q2-2023-Checklist#images), we should also have a `screenshot.svg` as an asset. Is that something that you provide? Or do I have take the screenshot.png file and somehow convert it into an SVG?

@eyevana I have to create it, but can you please track down any specs if possible? It has been a long while since I created one.

**Comment by eyevana** [comment]
> > @coreyvickery according to the docs cleanup [guide](https://github.com/RedHat-UX/red-hat-design-system/wiki/Docs-Cleanup-Q2-2023-Checklist#images), we should also have a `screenshot.svg` as an asset. Is that something that you provide? Or do I have take the screenshot.png file and somehow convert it into an SVG?
> 
> @eyevana I have to create it, but can you please track down any specs if possible? It has been a long while since I created one.

Nevermind. I see a screenshot.svg file now. Looks like Steven added it a week or two ago.

**Comment by coreyvickery** [comment]
@eyevana Couple of tiny edits and then we are done:

### Style tab
- Under the `Theme` h2, `Light theme` and `Dark theme` need to be h3

### Guidelines tab
- Under the `Layout` h2, some of the images look stretched; I forget, are waiting on another issue to fix this once and for all?
- Under the `Responsive design` h2, same feedback as above
- Under the `Responsive design` h2, can you change the styles of the screen size text (`Desktop`, `Tablet`, etc.) to use the same styles at the text under the `Anatomy` image in the `Style` tab? (Red Hat Text, 14px, 1.5/21 line height, #4D4D4D)

**Comment by eyevana** [comment]
@coreyvickery alrighty, I think we should be good now! :) Take a look and let me know what you think (pleaseee).

**Review by coreyvickery — APPROVED** [review]
Looks great!

File: elements/rh-pagination/docs/20-guidelines.md [hunk 1]
```diff
@@ -102,13 +102,13 @@ Truncation does not apply to the Compact size because it does not display page n
 
 Both pagination sizes and orientations are horizontally centered below content by default.
 
-![Image of full size pagination with the page input field on the right]({{ '../pagination-alignment-pif-right-a.png' | url }})
+![Image of full size pagination with the page input field on the right]({{ '../pagination-alignment-pif-right-a.png' | url }}){style="--inline-img-max-width:1000px;"}
```
Comment (bennypowers) [inline elements/rh-pagination/docs/20-guidelines.md:105]: We should probably use example shortcode instead of these css props

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: elements/rh-pagination/docs/20-guidelines.md (hunk 1 above)
Comment (eyevana) [reply elements/rh-pagination/docs/20-guidelines.md]: Done. I switched all of the inline images to the example shortcode with a palette of none.

**Review by bennypowers — APPROVED** [review]

**Comment by coreyvickery** [comment]
@eyevana Let's open a new issue for these small changes. @bennypowers Feel free to tackle this if you want.

### Guidelines page
- Under the `Sizes` h2, looks like the bullet line spacing is looser than normal and also the image in the container is indented
- Under the `Responsive design` h2 and under the `With page input field` h3, delete the images/text in between the `<p>` tags and replace with the image `pagination-page-input-field-right.png`
- Under the `Responsive design` h2 and under the `If the page input field is used below,` `<p>` tag, delete the images/text and replace with the image `pagination-page-input-field-below.png`
- Under the `Responsive design` h2 and under the `Without page input field` h3, delete the images/text and replace with the image `pagination-without-page-input-field.png`

**Comment by eyevana** [comment]
@coreyvickery No worries. I got this! Thank you.

**Pull request merged**